src/components/UploadForm/style.scss

```diff
@@ -0,0 +1,10 @@
+.button-container {
```

Comment:
just transfer all the `stylesVarsHooks.jsx` styles into here, why use a different one file